app/waf/src/models/common.model.ts

```diff
@@ -1,6 +1,15 @@
 import {Entity, property} from '@loopback/repository';
+import {MetadataInspector} from '@loopback/metadata';
+import {MODEL_PROPERTIES_KEY, PropertyDefinition} from '@loopback/repository';
+import {AS3Declaration} from '.';
+
+function as3Name(postfix: string) {
```

Comment:
It changes the user's given names and unrecoverable. 
We may need to find a way to solve it, etc. encoding with base64 or so..

Reply:
This property is the index key of AS3 object, not the object name given by user. But I think change that index key to something like base64 encoded string seems to be a good idea.

Reply:
AS3 name does not accept '=', we need another way to encode..

Reply:
I change the AS3 name back to 'F5_XXX'. And input resource id to AS3 object label property. Let us look for a better encoding method later.